### ISSUE TITLE
Input: `has-error` and `is-disabled` classnames should be added to the container

### ISSUE THREAD

**Issue by edizcelik** [Hipo/react-ui-toolkit#107]
Currently they are just added to the Input. However, to make the component more customizable, we need to add these classes to the container. For example, a user of this component may want to change the color of an icon provided to the component (as leftIcon or rightIcon).

**Comment by yasincaliskan** [comment]
Of course! So, should it be `input-container--has-error` and `input-container--is-disabled`?

**Comment by edizcelik** [comment]
Yes.
